app/components/NavModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const NavModalDialog = styled.div.attrs({
+  className: 'modal-dialog',
+})`
+  &&& {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+`;
+
+const NavModalContent = styled.div.attrs({
+  className: 'modal-content',
+})`
+  &&& {
+    background: transparent;
+    border: transparent;
+
+    & > div {
```

Comment:
Can you explain this CSS?

Reply:
The children of modal-content with other modal Bootstrap classes will have their styling overwritten by the Bootstrap styles (removing transparency) unless there's an increased specificity in our own stuff. The & is arbitrary as it just repeats the parent specificity, but I think it's good to have there for readability. I made it specifically point to the direct children to increase specificity (as opposed to just being descendants by leaving out the angle bracket).